src/components/pseudonym/PseudonymCard.styles.scss

```
@@ -108,15 +108,15 @@
 	font-size: 14px;
 	font-weight: 600;
 	line-height: 16px;
-	color: #1f2937;
+	color: var(--m3-on-surface, #1f2937);
 }
 
 .pseudonymCard__headerSubtitle {
 	font-family: 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)